### PR TITLE
Add crashtest support to Safari

### PR DESCRIPTION
**Pull request by Hexcles**
This is basically the same as #20914 .

### CODE REVIEW

**Comment by Hexcles** [comment]
Also pushed to `triggers/safari_preview`: https://dev.azure.com/web-platform-tests/wpt/_build/results?buildId=45350&view=results

**Comment by Hexcles** [comment]
The unittest failure is unrelated (and being worked around in https://github.com/web-platform-tests/wpt/pull/22760).

File: infrastructure/crashtests/example.html [hunk 1]
```diff
@@ -1 +1,2 @@
+<!-- DO NOT SUBMIT -->
```
Comment (jgraham) [inline infrastructure/crashtests/example.html:1]: If you change wptrunner I think we run all the infra tests anyway.

**Review by jgraham — APPROVED** [review]
Please remove bogus changes, but otherwise lgtm.

File: infrastructure/crashtests/example.html (hunk 1 above)
Comment (Hexcles) [reply infrastructure/crashtests/example.html:1]: Ahh I forgot. Thanks!

**Pull request merged**